### ISSUE TITLE
[Troubleshoot]: GPU utilization periodically drops to near zero during RECAP value model SFT training

### ISSUE THREAD

**Issue by Renajvruo** [RLinf/RLinf#1248]
### Problem description

help~help~help~ I'm training a RECAP value model using the official example script.During training, I observe severe performance degradation over time: GPU utilization periodically drops to ~0% and stays low for seconds to minutes, GPU memory usage also drops to near zero during these idle periods.

### Configuration YAML file

# Usage:
#   bash examples/recap/value/run_value_sft.sh libero10_task0_value
#
# This config trains the RECAP value model on the Libero10 Task0 SFT + rollout
# datasets whose returns were generated by examples/recap/process/compute_returns.py.

defaults:
  - model/value@actor.model
  - training_backend/fsdp@actor.fsdp_config
  - _self_
  - override hydra/job_logging: stdout

hydra:
  run:
    dir: .
  output_subdir: null
  searchpath:
    - file://${oc.env:REPO_PATH}/examples/sft/config/

cluster:
  num_nodes: 1
  component_placement:
    actor,env,rollout: all

runner:
  task_type: sft
  logger:
    log_path: "../results"
    project_name: rlinf
    experiment_name: "libero10_task0_value_sft"
    logger_backends: ["tensorboard"]

  max_epochs: 30000
  max_steps: 30000
  val_check_interval: 3000
  save_interval: 3000

data:
  # Must match compute_returns.py's data.tag.
  # Your provided compute_returns config uses tag: null, so this loads:
  #   meta/returns.parquet
  # If you later use tag: recap_v1, change this to: recap_v1
  tag: null

  train_data_paths:
    - dataset_path: "/mnt/cfs/data/data/recapdata/RECAP-Libero10-Task0-48succ-Data/libero10_task0_sft"
      type: "sft"
      weight: 50
      robot_type: "libero"
      model_type: "pi05"

    - dataset_path: "/mnt/cfs/data/data/recapdata/RECAP-Libero10-Task0-48succ-Data/libero10_task0_train"
      type: "rollout"
      weight: 1.0
      robot_type: "libero"
      model_type: "pi05"

  # No separate eval dataset path was provided. Use rollout data for value eval
  # for now. Replace this with your real eval dataset if available.
  eval_data_paths:
    - dataset_path: "/mnt/cfs/data/data/recapdata/RECAP-Libero10-Task0-48succ-Data/libero10_task0_eval"
      max_samples: 10000
      robot_type: "libero"
      model_type: "pi05"

  # ----- dataloader performance -----
  train_num_workers: 8
  eval_num_workers: 4
  prefetch_factor: 4
  persistent_workers: true
  pin_memory: true

  # ----- shared dataset params -----
  include_state: true
  gamma: 1.0
  action_horizon: 10
  normalize_to_minus_one_zero: true
  include_next_obs: false
  action_dim: 7
  robot_type: "libero"
  model_type: "pi05"

  # ----- mixture settings -----
  balance_weights: true
  seed: 42

algorithm:
  adv_type: gae

actor:
  group_name: "ActorGroup"
  training_backend: "fsdp"
  micro_batch_size: 32
  global_batch_size: 256
  seed: 0

  model:
    precision: bf16
    siglip_path: "/mnt/cfs/data/recap/RLinf/premodel/siglip2-so400m-patch14-224"
    gemma3_path: "/mnt/cfs/data/recap/RLinf/premodel/gemma-3-270m"
    tokenizer_path: "/mnt/cfs/data/recap/RLinf/premodel/gemma-3-270m"
    freeze_vlm: false
    action_dim: 7
    action_horizon: 10

  optim:
    lr: 5.0e-5
    value_lr: 1.0e-4
    adam_beta1: 0.9
    adam_beta2: 0.95
    adam_eps: 1.0e-8
    weight_decay: 1e-10
    clip_grad: 1.0
    lr_scheduler: "constant"
    lr_warmup_steps: 500
    total_training_steps: 8000

  fsdp_config:
    strategy: "fsdp"
    sharding_strategy: "no_shard"
    use_orig_params: False
    gradient_checkpointing: true
    mixed_precision:
      param_dtype: ${actor.model.precision}
      reduce_dtype: ${actor.model.precision}
      buffer_dtype: ${actor.model.precision}

reward:
  use_reward_model: False

critic:
  use_critic_model: False

### Log file

<img width="1322" height="589" alt="Image" src="https://github.com/user-attachments/assets/aa342750-724f-4b06-88eb-f03b0ee44b56" />
<img width="1410" height="304" alt="Image" src="https://github.com/user-attachments/assets/c23a0f39-e318-4df9-a1a2-048775145f57" />

### Environment

python -V:
uv pip list:
nvidia-smi:
RLinf version:
Docker image tag: 

### Before submitting a new issue...

- [x] Have you checked relevant issues, FAQs (https://rlinf.readthedocs.io/en/latest/rst_source/faq.html), or asked the chatbot at the top right corner of the [documentation page](https://rlinf.readthedocs.io).

**Comment by Nadeem752** [comment]
@Renajvruo how are you planning on running the inference ?  the infference script dosent support RECAP

**Comment by Renajvruo** [comment]
> [@Renajvruo](https://github.com/Renajvruo) how are you planning on running the inference ? the infference script dosent support RECAP

hi，I plan to modify the original openpi repository code to add support for RECAP's configuration (cfg) during training, so that the inference script can properly run models related to RECAP."

**Comment by Rkdamawang** [comment]
@Renajvruo Hello, I'd like to ask, what is your GPU device? How long have you been training?
